### PR TITLE
Add scrolling and enhance list of affected detectors

### PR DESCRIPTION
**Pull request by ohltyler**
*Issue #, if available:* #193 

*Description of changes:*

This PR changes 2 things about the batch action modals:
1) Adds scrolling functionality to prevent the list of detectors/monitors to be truncated.
2) Changes layout of the list to be an `EuiDataGrid` for cleaner formatting and readability.

Notes:
- Removed the styling added for the `EuiModalBody__overflow` component. Instead, added an `EuiSpacer` in the `ConfirmModal` component to prevent the modal from being in an overflow state (see [this line](https://github.com/elastic/eui/blob/3a8c0a903d889b85e77f71ce5b3c57089838f469/src/components/modal/_modal.scss#L25)).
- Confirmed existing modals don't unnecessarily show the scrollbar
- Confirmed all UT pass

Screenshot (scrolling):
Screen Shot 2020-06-05 at 1 06 08 PM

Screenshot (start batch action):
Screen Shot 2020-06-05 at 1 06 47 PM

Screenshot (stop batch action):
Screen Shot 2020-06-05 at 1 06 27 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by ohltyler** [comment]
@ylwu-amzn @yizheliu-amazon scrolling issue fixed by the way, please read PR description for more

File: public/pages/DetectorsList/Components/ConfirmActionModals/ConfirmActionModals.tsx [hunk 1]
```diff
@@ -43,98 +55,210 @@ export const ConfirmStartDetectorsModal = (
...
 ) => (
   <EuiOverlayMask>
-    <ConfirmModal
-      title="Are you sure you want to stop the selected detectors?"
-      description=""
-      callout={
+    <EuiModal onClose={props.hideModal}>
+      <EuiModalHeader>
+        <EuiModalHeaderTitle>
+          {'Are you sure you want to stop the selected detectors?'}&nbsp;
```
Comment (yizheliu-amazon) [inline public/pages/DetectorsList/Components/ConfirmActionModals/ConfirmActionModals.tsx:99]: Do we have those wording reviewed by Ashwin?

Comment (ohltyler) [reply public/pages/DetectorsList/Components/ConfirmActionModals/ConfirmActionModals.tsx:99]: All wording has been approved by Carlos. Do I need Ashwin as well?

Comment (yizheliu-amazon) [reply public/pages/DetectorsList/Components/ConfirmActionModals/ConfirmActionModals.tsx:99]: just realized this is same as previous title, should be fine

Comment (yizheliu-amazon) [reply public/pages/DetectorsList/Components/ConfirmActionModals/ConfirmActionModals.tsx:99]: > All wording has been approved by Carlos. Do I need Ashwin as well?

For such short ones, it should be fine. nvm.

**Pull request merged**